lab/src/Microsoft.ServiceModel.Syndication/src/Rss20Constants.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Microsoft.ServiceModel.Syndication
 {
-    internal static class Rss20Constants
+    public static class Rss20Constants
     {
```

Comment:
I thought it would be nice to have it public because when the user has to use the custom parser, he will need to know the exact tag that we are using to parse.

---
In reply to: [125780076](https://github.com/dotnet/wcf/pull/2028#discussion_r125780076) [](ancestors = 125780076)